### PR TITLE
libpod/runtime.go: runtime path

### PR DESCRIPTION
**Pull request by baude**
Ubuntu installs runc to /usr/sbin/runc so we now account
for that.  Also, added  small check when creating a new
runtime that if we cannot find the runc binary, we bail
out.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by baude** [comment]
@mheon PTAL

File: libpod/runtime.go [hunk 1]
```diff
@@ -101,6 +101,15 @@ func findConmonPath() string {
 	return path
 }
 
+func findRuncPath() string {
+	path := "/usr/sbin/runc"
```
[the remark this answers is not in the document]
Comment (baude) [reply libpod/runtime.go]: ok,fixed

**Comment by mheon** [comment]
LGTM

**Comment by baude** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**